elements/pl-matrix-output/pl-matrix-output.py

```diff
@@ -45,13 +59,27 @@ def render(element_html, data):
                 prefix = 'np.array('
                 suffix = ')'
 
+            # Mathematica reserved letters: C D E I K N O
+            mathematica_reserved = ['C', 'D', 'E', 'I', 'K', 'N', 'O']
+            if pl.inner_html(child) in mathematica_reserved:
+                mathematica_suffix = 'm'
+            else:
+                mathematica_suffix = ''
+
             # Create string for matlab and python format
-            matlab_data += pl.inner_html(child) + ' = ' + pl.string_from_2darray(var_data, language='matlab', digits=digits) + ';\n'
-            python_data += pl.inner_html(child) + ' = ' + prefix + pl.string_from_2darray(var_data, language='python', digits=digits) + suffix + '\n'
+            if commentExists == 0:
+                matlab_data += pl.inner_html(child) + ' = ' + pl.string_from_2darray(var_data, language='matlab', digits=var_digits) + ';\n'
+                mathematica_data += pl.inner_html(child) + ' = ' + pl.string_from_2darray(var_data, language='matlab', digits=var_digits) + ';\n'
+                python_data += pl.inner_html(child) + ' = ' + prefix + pl.string_from_2darray(var_data, language='python', digits=var_digits) + suffix + '\n'
+            else:
+                matlab_data += pl.inner_html(child) + ' = ' + pl.string_from_2darray(var_data, language='matlab', digits=var_digits) + '; % ' + var_comment + '\n'
+                mathematica_data += pl.inner_html(child) + mathematica_suffix + ' = ' + pl.string_from_2darray(var_data, language='matlab', digits=var_digits) + '; (* ' + var_comment + ' *)\n'
+                python_data += pl.inner_html(child) + ' = ' + prefix + pl.string_from_2darray(var_data, language='python', digits=var_digits) + suffix + ' # ' + var_comment + '\n'
 
     html_params = {
         'default_is_matlab': True,
```

Comment:
Is this hard-coded? I thought we got to choose?

Reply:
It has been been hard-coded, and I agree that we should choose! It is okay to have just another attribute :-)

Reply:
Yep, an attribute for this would be great!

Reply:
What do you mean hardcoded? Choosing which tabs appear?

Reply:
@eliving2 I think they're discussing what should be the "default" tab shown on question load. Within `html_params`, only ` 'default_is_matlab': True` is set. However, the mustache template has: 

-`#default_is_matlab`
-`#default_is_python`
- and now `#default_is_mathematica`

Reply:
Ended up implementing an option to hide tabs. No idea how I would change the display order in the mustache file. If someone knows how to implement this, I am happy to fix it

Reply:
Added a default-tab option, resolving conversation.